packages/office-ui-fabric-react/etc/office-ui-fabric-react.api.ts

```diff
@@ -10556,7 +10557,8 @@ interface ISpinButtonProps {
 // @public (undocumented)
 interface ISpinButtonState {
   isFocused: boolean;
-  keyboardSpinDirection: KeyboardSpinDirection;

```

Comment:
Do not remove the old thing and rename it; this will break everyone. Instead, keep the old one, mark it as `@deprecated`, and add the new one. this gives partners an opportunity to fix the code.

Reply:
We also tend to write tests in test.deprecated.ts files, to make sure the old still works. We separate out "deprecation" tests so that they're easy enough to clean up later.

Reply:
This is a state rename - to remove the `keyboard` prefix because this state represent both the `keyboard` and `mouse` case now. I don't understand why this is considered as a part of public API.

If we want to keep the backward compatibility. The change is to rename the variable back to `keyboardSpinDirection` and mark it as `@deprecated`, then create another shadow state named `spinDirection`.  There is no test on the original state.